src/patternfly/demos/Toolbar/examples/toolbar-complex-example.hbs

```diff
@@ -0,0 +1,99 @@
+{{#> toolbar }}
+    {{#> toolbar-section pf-l-toolbar--modifier="pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md" aria-label="First section"}}
+        {{#> toolbar-group}}
+            {{#> toolbar-item pf-l-toolbar__item--modifier="pf-u-mr-2xl"}}
+                {{#> input-group }}
+                    {{#> dropdown id="dropdown-example-collapsed"}}
+                        All
+                        <i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i> 
+                    {{/dropdown}}
+                {{#> input-group-action input-group-action--modifier="pf-m-align-right" inputGroupTextType="label" aria-label="popover for input"}}
+                    <i class="fas fa-search aria-hidden="true"></i>
+                {{/input-group-action}}
+                {{#> form-control controlType="input" input="true" type="text" id="textInput10" name="textInput10" placeholder="Filter..." aria-label="input example with popover" }}
```

Comment:
Can you update the aria-label here to say "filter input with popup button"

Reply:
fixed